backend/core/test/assets/assets.e2e-spec.ts

```diff
@@ -70,7 +70,7 @@ describe("AssetsController", () => {
   describe("retrieve", () => {
     it("should return a paginated assets list", async () => {
       const res = await supertest(app.getHttpServer())
-        .get(`/assets/`)
+        .get(`/assets?page=1&limit=10`)
```

Comment:
What makes this request need a page and limit?

Reply:
Good question! And it made me realize that what I actually wanted to do was make `GET /assets` resilient to the case where we set `page` and/or `limit` to -1. So I reverted the change to the test and instead change `assets.service.ts` to look like the applications service: https://github.com/CityOfDetroit/bloom/blob/da16a46128e20c1ebdeb999e2edfe41a81f22d97/backend/core/src/applications/applications.service.ts#L51